RecoverVm/RecreateVmFromVhd.ps1

```diff
@@ -0,0 +1,90 @@
+﻿function RecreateVmFromVhd (
+    [string] $ServiceName,
+    [string] $RecoVmName,
+    [bool] $DeleteRecoVM
+    )
+{
+    
+    $attachedVm = Get-AzureVM -ServiceName $ServiceName -Name $RecoVmName
+    $location = (Get-AzureService -ServiceName $ServiceName).Location
+    if ( $attachedVm.VM.DataVirtualHardDisks.Count -eq 0 )
+    {
+        Write-Error "No data disk attached to recover vm $RecoVmName unable to procede!"
+        return
+    }
+
+    $OSDisk = $attachedVm.VM.DataVirtualHardDisks[0]
```

Comment:
We assume that the recovered disk is the only/first data disk.  Is that a valid assumption?  Do we need to add this assumption to the readme?